File: Comparativa Python/Ejercicio6-Tablas.py
# Realice un programa que calcule la suma de dos matrices cuadradas de 3 x 3
import numpy as np

m1 = np.zeros((3, 3), dtype=int)
m2 = np.zeros((3, 3), dtype=int)

# Ingresar datos en matriz1

for r in range(3):
    for c in range(3):
        m1[r][c] = int(input('Digite un número M1 [{}][{}]: '
                             .format(r+1, c+1)))

# Ingresar datos en matriz2

for r in range(3):
    for c in range(3):
        m2[r][c] = int(input('Digite un número M2 [{}][{}]: '
                             .format(r+1, c+1)))

print('La matriz 1: \n', m1)
print('La matriz 2: \n', m2)
print('La suma de matricez es: \n', m1 + m2)
"""
for r in range(3):
    for c in range(3):
        print(m1[r][c], ' ', end='')
    print(' ')
"""

File: Matrices o Tablas/Ejercicio6-Tablas.cpp
/*Realice un programa que calcule la suma de dos matrices cuadradas de 3 x 3*/

#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    int matriz1[3][3];
    int matriz2[3][3];
    int matrizR[3][3];

    for (int i=0; i<3; i++){
        for(int j=0; j<3; j++){
            cout<<"Digite un numero M1 ["<<i+1<<"]["<<j+1<<"]: ";
            cin>>matriz1[i][j];
        }
    }
    for (int i=0; i<3; i++){
        for(int j=0; j<3; j++){
            cout<<"Digite un numero M2 ["<<i+1<<"]["<<j+1<<"]: ";
            cin>>matriz2[i][j];
        }
    }
    cout<<"Mostrando las dos matrices:\n";

    for(int i=0;i<3;i++){
        for(int j=0;j<3;j++){
            cout<<matriz1[i][j]<<" ";
        }
        cout<<endl;
    }
    cout<<endl;

    for(int i=0;i<3;i++){
        for(int j=0;j<3;j++){
            cout<<matriz2[i][j]<<" ";
        }
        cout<<endl;
    }

    for (int i=0; i<3; i++){
        for(int j=0; j<3; j++){
            matrizR[i][j]=matriz1[i][j]+matriz2[i][j];
        }
    }
    cout<<"La matriz resultado es:"<<endl;

    for (int i=0; i<3; i++){
        for(int j=0; j<3; j++){
            cout<<matrizR[i][j]<<" ";
        }
        cout<<endl;
    }

    system("pause");
    return 0;
}